go/pgn/main.go

```
@@ -103,6 +106,11 @@ func realMain() int {
 	}
 	defer sqlite.Db.Close()
 
+	go func(ctx context.Context, pgnId string, ia addr.IA, plnIA addr.IA) {
+		defer log.HandlePanic()
+		plncomm.AddPGNEntry(ctx, pgnId, ia, plnIA)
```

>plncomm.AddPGNEntry

The error is not handled.